.testcaferc.json

```diff
@@ -1,6 +1,17 @@
 {
   "browsers": [
-    "chrome"
+    "chrome:headless"
   ],
-  "src": "test/testcafe/spec/"
+  "clientScripts": [
+    {
+      "module": "axe-core/axe.min.js"
+    }
+  ],
+  "concurrency": 4,
+  "src": [
+    "test/testcafe/spec/*_spec.js"
+  ],
+  "userVariables": {
+    "v3": false
+  }
```

Comment:
Is this needed in branches from the mainline?

Reply:
Nope. Will clean up